src/Augurk.Test/Managers/DependencyManagerTests.cs

```diff
@@ -1,37 +1,31 @@
-﻿using System;
-using System.Text;
-using System.Collections.Generic;
+﻿using System.Collections.Generic;
 using Microsoft.VisualStudio.TestTools.UnitTesting;
-using Raven.Client.Embedded;
 using Augurk.Api;
 using Augurk.Api.Managers;
 using System.Threading.Tasks;
 using System.Linq;
+using Raven.TestDriver;
+using Raven.Client.Documents;
 
 namespace Augurk.Test.Managers
 {
     /// <summary>
     /// Summary description for DependencyManagerTests
     /// </summary>
     [TestClass]
-    public class DependencyManagerTests
+    public class DependencyManagerTests : RavenTestDriver
```

Comment:
RavenTestDriver seems to be the recommended way to do testing with RavenDB now. In the DI branch I've moved much of the setup into an abstract base class that can be used for tests. I think we should invest in some more tests either way, but I think this is clean enough for now.